src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -6,21 +6,26 @@ The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\ma
 struct NormInfinityBridge{T} <: AbstractBridge
     ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
 end
-function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormInfinityCone) where T
-    return bridge_constraint(NormInfinityBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
-end
-function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorAffineFunction{T}, s::MOI.NormInfinityCone) where T
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.AbstractVectorFunction, s::MOI.NormInfinityCone) where T
     f_scalars = MOIU.eachscalar(f)
     t = f_scalars[1]
+    if t isa MOI.SingleVariable # TODO currently needed for operate_output_index!
+        t = MOI.ScalarAffineFunction{T}(t)
+    end
     d = MOI.dimension(s)
-    new_f = MOIU.vectorize(vcat([MOIU.operate(-, T, t, f_scalars[i]) for i in 2:d], [MOIU.operate(+, T, t, f_scalars[i]) for i in 2:d]))
-    ge_index = MOI.add_constraint(model, new_f, MOI.Nonnegatives(2d - 2))
+    lb = f_scalars[2:d]
+    ub = MOIU.operate(-, T, lb)
+    f_new = MOIU.operate(vcat, T, ub, lb)
+    for i in 1:MOI.output_dimension(f_new)
+        MOIU.operate_output_index!(+, T, i, f_new, t)
```

Comment:
@blegat is this how you envisioned it?

Reply:
Yes, I like that! Maybe you are missing a second line `MOIU.operate_output_index!(+, T, d + i, f_new, t)` for the `lb` part ?

Reply:
on line 18 (before this) I vcat ub and lb

Reply:
Yes but then you only add `t` to the indices corresponding to `ub`, so `t` is not constrained to be higher than the lower bound, you simply have `-lb <= 0`

Reply:
I did it differently to how you suggested - on line 18, I vcat to get `f_new`, then I add `t` to every element of `f_new`, which is double the length of ub

Reply:
the loop is `for i in 1:MOI.output_dimension(f_new)`

Reply:
> the loop is `for i in 1:MOI.output_dimension(f_new)`

Aaaaah, that's what I was missing, I thought it was `for i in 1:d`